orc8r/cloud/go/tools/combine_swagger/spec/combine_test.go

```diff
@@ -50,16 +51,19 @@ func TestCombine(t *testing.T) {
 	cleanup()
 	defer cleanup()
 
-	cfgs, common, err := spec.Load(cfgsDir, commonFilepath)
+	specs, commonSpec, err := spec.Load(cfgsDir, commonFilepath)
```

Comment:
Leaving this as `specs, common, err` is sufficient / prioritizes brevity, which has added desirability in tests